core/src/main/java/org/wildfly/plugin/core/GalleonUtils.java

```diff
@@ -247,9 +248,9 @@ private static String getMavenCoords(FeaturePack fp) {
         return builder.toString();
     }
 
-    private static String getWildFlyFeaturePackLocation(String version) {
+    private static String getWildFlyFeaturePackLocation(String featurePackLocation, String version) {
         StringBuilder fplBuilder = new StringBuilder();
-        fplBuilder.append(WILDFLY_DEFAULT_FEATURE_PACK_LOCATION);
+        fplBuilder.append(featurePackLocation);
```

Comment:
We could throw an exception if featurepackLocation is null.

Reply:
Good point. When I tested manually it did error out, but being more explicit probably makes more sense. I wonder if we should validate the pattern too.